atc/api/pipelineserver/build.go

```diff
@@ -22,22 +21,13 @@ func (s *Server) CreateBuild(pipelineDB db.Pipeline) http.Handler {
 			return
 		}
 
-		build, err := pipelineDB.CreateOneOffBuild()
+		build, err := pipeline.CreateStartedBuild(plan)
```

Comment:
kicking the build tracker here would be nice

Reply:
LISTEN/NOTIFY work?